src/mono/nuget/mono-packages.proj

```diff
@@ -18,6 +18,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetsMobile)' == 'true'">
+    <ProjectReference Include="Microsoft.NET.Runtime.MonoAOTCompiler.Workload\Microsoft.NET.Runtime.MonoAOTCompiler.Workload.pkgproj" />
```

Comment:
I don't think we should add another nuget. The manifest will be always coupled with SDK so we should just add the file to SDK.

Reply:
That's right...  In that case, we should hard code a baseline version, correct?

Reply:
We should not hardcode it but let it rev with SDK automatically.